wallet/wallet.sol

```diff
@@ -337,7 +337,9 @@ contract Wallet is multisig, multiowned, daylimit {
     // and _data arguments). They still get the option of using them if they want, anyways.
     function execute(address _to, uint _value, bytes _data) external onlyowner returns (bytes32 _r) {
         // first, take the opportunity to check that we're under the daily limit.
-        if (underLimit(_value)) {
+        // we also must check that there is no data (this is not a contract invocation),
```

Comment:
Every transaction executes code, even if the data is empty.
You only be sure that no code will execute by checking that `extcodesize == 0`:
```
function hasCode(address _addr) returns (bool) {
  uint size;
  assembly { size := extcodesize(_addr) }
  return size > 0;
}
```
Untested code, though.